blog/2024/09-10-release-roadmap-10.10.0/index.mdx

```diff
@@ -0,0 +1,81 @@
+---
+title: Release Roadmap for 10.10.0
+description: We are about to start the 10.10.0 release cycle; here is the roadmap. Please help us test!
+authors: joshuaboniface
+slug: release-roadmap-10.10.0
+tags: [release, testing]
+---
+
+![Good heavens, look at the time!](./10.10.0.jpg)
+
+We are pleased to announce that we are now beginning the process for the 10.10.0 release, with a *planned* release date of Saturday, October 12th, 2024! We said 6 months in our 10.9.0 release posts, and we're sticking by that! The feature freeze proper starts next week, with a soft "new PR freeze" today, so here's what you need to know about the timeline and a recap of how to help us test 10.10.0 before release. Remember, the more people who help test it out before release, the less bugs we're likely to find *after* release, so fire up those secondary servers and warn your users: 10.10.0 is coming!
```

Comment:
```suggestion
We are pleased to announce that we are now beginning the process of the 10.10.0 release, with a *planned* release date of Saturday, October 12th, 2024! We said 6 months in our 10.9.0 release posts, and we're sticking by that! The feature freeze proper starts next week, with a soft "new PR freeze" today, so here's what you need to know about the timeline and a recap of how to help us test 10.10.0 before release. Remember, the more people who help test it out before release, the less bugs we're likely to find *after* release, so fire up those secondary servers and warn your users: 10.10.0 is coming!
```

Reply:
I prefer "for" here as its a process towards the release, "of" doesn't feel right.